.github/env/keycloak-rel-testing

```diff
@@ -4,4 +4,7 @@ mvn-releases-url=https://maven.pkg.github.com/keycloak-rel-testing/maven
 mvn-snapshots-url=https://maven.pkg.github.com/keycloak-rel-testing/maven
 
 quay-org=keycloaktesting
-docker-org=keycloaktesting
\ No newline at end of file
+docker-org=keycloaktesting
+
+community-operators-repo=https://github.com/keycloak-bot/community-operators.git
+prod-operators-repo=https://github.com/keycloak-bot/community-operators-prod.git
```

Comment:
The bot's repo should act as the originating fork that will be used for the PRs creation, not a target.
```suggestion
community-operators-repo=https://github.com/keycloak-rel-testing/community-operators.git
prod-operators-repo=https://github.com/keycloak-rel-testing/community-operators-prod.git
```

Reply:
@vmuzikar Yep, probably you're right. I thought it's gonna be sufficient to have only forks for `keycloak-rel` and `keycloak-bot` in order to save some "maintenance" points. In this case, we need to create 6 forks; community, and prod versions of these operators for each "organization".

Reply:
There will be no fork in `keycloak-rel`. During the real release, well use the bot's fork to create PRs in the end.

Reply:
Oh, you're right!